sandbox/data/default.js

```diff
@@ -90,7 +90,19 @@ module.exports = {
         },
         {
             source: 'VUX',
-            target: 'Intruder'
+            target: 'Intruder 1'
```

Comment:
It might be useful to have a specific data set (for instance **collapsible**) for this feature, this can be done by doing the following:
1. Create a folder under `sandbox/data/collapsible`
2. Add a config and a data file where you can tweak the data and configs in a more convenient way (collapsible.config.js and collapsible.data.js)
3. Load the dataset in the sandbox by adding a query parameter to the url: `?data=collapsible`

Reply:
Sounds like an excelent idea! I can create another PR with this feature included without a doubt.

Reply:
Maybe this could be useful for this PR in order for us to do some functional tests on top of the dataset.

Reply:
You are correct, haven't thought about the functional tests in this context.